infrastructure/lookerstudio/README.md

```diff
@@ -1,7 +1,16 @@
-# Marketing Analytics Looker Studio Dashboard
+# Marketing Data Engine Looker Studio Dashboard
 
+This Python script generates a URL to deploy a copy of the Marketing Data Engine dashboard utilizing the [Looker Studio Linking API](https://developers.google.com/looker-studio/integrate/linking-api).
 
+The dashboard deployment is intended to be completed once the Marketing Data Engine warehouse has been deployed through Dataform.
+
+The libraries required are in requirements.txt.
+
+You will need to configure config.ini with the BigQuery project and datasets for your warehouse.
 
 ## Run the walkthrough tutorial
 
+Click below to open the deployment tutorial in the Google Cloud console.
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/TXZebra/marketing-data-engine&cloudshell_tutorial=infrastructure/lookerstudio/lookerstudio-deployment-walkthrough.md)
```

Comment:
You cannot use the cloudshell_git_repo=...

The issue is that you gonna have to define the Github Repo owner (TXZebra) and the Github forked repo name (marketing-data-engine) as a fixed values, so I would not suggest that path.

Reply:
I forgot to change the link to the GoogleCloudPlatform repo, I can update this and it should work as expected. Are you suggesting not to use the Cloud Shell tutorial and instead have them follow the Readme directly in GitHub instead? I originally chose to use the Cloud Shell tutorial as a way to guide them through the deployment within the console but if we prefer not to go that route, I can modify.

Reply:
Thinking about this more, if we need to use poetry then the Cloud Shell Tutorial won't work in the way intended. I will work to convert everything to be done within GitHub directly.

Reply:
No, I don't think you get it yet. Don't use the Cloud Shell Tutorial because people are gonna need to fork optionally. We don't want to enforce people use follow a Cloud Shell Tutorial. They can follow the instructions on README directly. Clearer?

Reply:
My response was prior to our conversation yesterday, I understand. Purely for discussion sake, forking should be immaterial as our repo will still contain the info but understood and changes have been made.